index.js

```diff
@@ -93,6 +93,23 @@ var GPS = function (hardware, callback) {
     }
   }, 1000);
 
+  // poll every 2 seconds by default, can bump this up but it'll consume more cpu
+  var pollTime = 2000;
+  self.getPollTime = function () {
+    return pollTime;
+  };
+  self.setPollTime = function (time) {
+    time = parseInt(time, 10);
+    if (time < 2000) {
+        // No less than 2 seconds
```

Comment:
Can you add a console.log that just spits out something like "GPS Poll time cannot be set less than 2 seconds, defaulting to 2 seconds."?

Just don't want the user confused about why it's not polling constantly if they set a low value.